ecs/testdata/simple/simple-cloudformation-conversion.golden

```diff
@@ -30,7 +30,7 @@ Resources:
       - Key: com.docker.compose.project
         Value: TestSimpleConvert
       - Key: com.docker.compose.network
-        Value: default
+        Value: TestSimpleConvert_default
```

Comment:
Side effect on ECS volumes, this seems better than it was before (using the actual volume name rather than yaml key), just checking this sounds a sound change @ndeloof